ci/constants.py

```diff
@@ -38,6 +38,13 @@ class TargetPython(Enum):
     'm2crypto',
     'netifaces',
     'Pillow',
+    # depends on cffi that still seems to have compilation issues
+    'protobuf_cpp',
+    'xeddsa',
+    'x3dh',
+    'pynacl',
+    'doubleratchet',
+    'omemo',
```

Comment:
@AndreMiras now that you have added autotools to the docker image, I guess I can remove those lines, right?

Reply:
Yes good remark, but well, it's complicated :joy: 
The libffi is fixed, but cffi still seems to fail for some reason. I didn't get time to investigate and I didn't want to postpone your merge further more.